src/test/test-locale-util.c

```diff
@@ -24,16 +24,11 @@ static void test_get_locales(void) {
 static void test_locale_is_valid(void) {
         log_info("/* %s */", __func__);
 
-        assert_se(locale_is_valid("en_EN.utf8"));
-        assert_se(locale_is_valid("fr_FR.utf8"));
-        assert_se(locale_is_valid("fr_FR@euro"));
-        assert_se(locale_is_valid("fi_FI"));
```

Comment:
i still thinkg @keszybz' suggestion should be implemented, to superfically validate that our expectations of what the glibc API does match reality